### ISSUE TITLE
Verify and brush in voxels on sides (around and bottom) of torso that weren't labelled as tissue by TotalSegmentator

### ISSUE THREAD

**Issue by mathieuboudreau** [shimming-toolbox/b0-fieldmap-realistic-simulation#16]
eg

![Screenshot 2024-11-04 at 10 19 58 AM](https://github.com/user-attachments/assets/7e5fe884-8703-4531-8c60-2d873363b903)

**Comment by mathieuboudreau** [comment]
If these aren't filled in, they will be assigned as air and will create pockets of air when padding

**Comment by mathieuboudreau** [comment]
Could also be due to this kind of "roundness" near the bottom of the torso / bumpy; maybe manually smoothing and stretching it to the boundary would improve the red-blue gradient in some maps.

![Screenshot 2024-11-04 at 10 29 58 AM](https://github.com/user-attachments/assets/15e2c47d-8e3e-4938-b876-6b3d6ea5d5b0)
![Screenshot 2024-11-04 at 10 29 44 AM](https://github.com/user-attachments/assets/51f31e92-1366-4222-ba45-28179e272252)

**Comment by mathieuboudreau** [comment]
Hmm, a "good" subject also has this and wasn't as much of an issue far from there,

![Screenshot 2024-11-04 at 10 33 01 AM](https://github.com/user-attachments/assets/2ba50c94-5fc8-49a1-b148-2793278269b5)

**Comment by mathieuboudreau** [comment]
And then, because we're demodulating for the tissue B0 values, the subjects that have "stronger b0 gradients" will have more (higher) red values, driving the mean value up, and thus making the neck/head of the subjects darker blue, eg

![Screenshot 2024-11-04 at 9 58 36 AM](https://github.com/user-attachments/assets/f738060a-544d-430c-8432-3ae0cce31c65)

instead of red-ish from the subjects that don't have this lung-induced gradient in the torso.

@jcohenadad maybe I should write two sentences on this in the abstract results section? It's not that the "head/neck" have more deviation towards negative B0 (blue) values for some subjects, but that the mean B0 changed due to higher lung asymmetry which then induced more B0 variation between/in the torso. Does this make sense? Pinging @evaalonsoortiz  too

**Comment by mathieuboudreau** [comment]
Here's a comparison of the B0 map for an axial slice for a "good" (sub 3 above) and a "bad" subject (sub 4 above)

# "Good" sub 3

![Screenshot 2024-11-04 at 11 18 44 AM](https://github.com/user-attachments/assets/93aa47fe-07ce-4cd7-b3d5-f37a50fb135b)


# "Bad" sub 4

![Screenshot 2024-11-04 at 11 17 46 AM](https://github.com/user-attachments/assets/fb11ced1-e742-447d-9140-706e2bc8ed0b)

They both seem to "touch" the walls of the FOV as much. Not sure if I can see another explanation for this apparent B0 gradient between the lungs, @evaalonsoortiz anything you see here?

**Comment by mathieuboudreau** [comment]
So if I go even lower, the furthest I can go before the "good" subject's left lung starts to disapear is this:

![Screenshot 2024-11-04 at 11 28 27 AM](https://github.com/user-attachments/assets/eae89652-a6db-4602-96d1-2b1d2834c683)

Whereas the "bad" subject, it's here:

![Screenshot 2024-11-04 at 11 28 08 AM](https://github.com/user-attachments/assets/459a0de6-63ff-44b8-8158-9475240fcc29)

So maybe it's more about the symmetry in length than thickness/thinness; for the "bad ones" there are a lot of slices where there's left lung but no/little right lung, whereas the "good" subject's are pretty symmetric

**Comment by evaalonsoortiz** [comment]
I agree with your explanations (both wrt to the left/right symmetry issue along the length of the spinal cord, and within the axial plane). Essentially, I think what this is all suggesting is that we should explore the possibility of acquiring anatomical scans with FOVs that encompass the whole torso (capturing the entirety of the lungs and their unique shapes seems important). If that is not doable, then perhaps in the future we should search for a large CT-based dataset that we can use for B0 simulations.

**Comment by mathieuboudreau** [comment]
The other problem is that for all images, the lower torso has really, really low SNR. So, the lower lung segmentations performed more or less ok there, decent but not phenomenal,

<img width="303" alt="Screenshot 2024-11-04 at 12 13 49 PM" src="https://github.com/user-attachments/assets/7f7a83ff-2b2f-47af-b7e0-c02e5f8c3bd8">
<img width="303" alt="Screenshot 2024-11-04 at 12 13 42 PM" src="https://github.com/user-attachments/assets/c298a9e8-9877-429e-9873-144d571ebf54">
